package.json

```diff
@@ -16,6 +16,7 @@
   "main": "SearchBar.js",
   "license": "MIT",
   "dependencies": {
+    "prop-types": "^15.5.10"
```

Comment:
missing comma, npm isn't going to like this

Reply:
Ooops

Reply:
Thanks 👍